metrics/meteor/meteor.py

```diff
@@ -113,16 +121,40 @@ def _download_and_prepare(self, dl_manager):
 
     def _compute(self, predictions, references, alpha=0.9, beta=3, gamma=0.5):
         if NLTK_VERSION >= version.Version("3.6.5"):
-            scores = [
-                meteor_score.single_meteor_score(
-                    word_tokenize(ref), word_tokenize(pred), alpha=alpha, beta=beta, gamma=gamma
-                )
-                for ref, pred in zip(references, predictions)
-            ]
+            if any(isinstance(el, list) for el in references):
```

Comment:
we only need to check the first element (if the types change from element to element there would be an error much earlier):
```suggestion
            if isinstance(references[0], list):
```